src/sample/trainers/forward_dynamics.py

```diff
@@ -0,0 +1,378 @@
+from collections.abc import Callable
+from functools import partial
+from pathlib import Path
+from typing import override
+
+import mlflow
+import torch
+from pamiq_core import DataUser
+from pamiq_core.torch import OptimizersSetup, TorchTrainer, get_device
+from pamiq_core.torch.trainer import TorchTrainer
+from torch import Tensor
+from torch.distributions import Distribution
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader, TensorDataset
+
+from sample.data import BufferName, DataKey
+from sample.models import ModelName
+from sample.models.forward_dynamics import ForwardDynamics
+from sample.trainers.sampler import RandomTimeSeriesSampler
```

Comment:
ここは同一モジュール内に対する参照なので相対参照を使います。
```suggestion
from .sampler import RandomTimeSeriesSampler
```

Reply:
https://github.com/MLShukai/pamiq-vrchat/pull/45/commits/2c7efac89f6e5a6095c28c93528bbe69f26f2bad
修正しました